src/SqlAccountRestAPI/Helpers/SqlAccountAppHelper.cs

```
@@ -256,4 +218,62 @@ public async Task<IDictionary<string, object>> Update()
             { "Status", "Update process started. Service will restart soon." },
         };
     }
+    public string GetVersion()
+    {
+        dynamic app = _factory.GetInstance();
+        return app.Version;
+    }
+    public async Task<IDictionary<string,object>> GetReleaseInfo(){
+        string configPath = await SystemHelper.GetCliConfigurationFilePath();
+        
+        if (!File.Exists(configPath))
+        {
+            return new Dictionary<string, object>
+                {
+                    { "Error", "The configuration file does not exist." }
+                };
+        }
+        try
+        {
+            var result = new Dictionary<string,object>{};
+            string fileContent = File.ReadAllText(configPath);
+
+            // Parse JSON to Dictionary
+            var options = new JsonSerializerOptions
+            {
+                AllowTrailingCommas = true,
+                ReadCommentHandling = JsonCommentHandling.Skip
+            };
+            var applicationInfo = JsonSerializer.Deserialize<Dictionary<string, object>>(fileContent, options);
+            // handle Github API limit
+            try {
+                var releaseInfo = await GithubHelper.GetLatestReleaseInfo();
+                applicationInfo!["LATEST_VERSION"] = releaseInfo["tag_name"];
+            }
+            catch (Exception ex){
+                applicationInfo!["LATEST_VERSION"] = $"Error: {ex.Message}";
+            }
+            
+            return applicationInfo;
+        }
+        catch (Exception ex)
```

đây nữa, ko work thì throw exception luôn. Cần gì phải build lại error message làm gì? Fail Fast anh đã nói nhiều lần rồi mà.